pxierrors.py

```diff
@@ -0,0 +1,151 @@
+"""
+Error classes for the PXI Server
+SaffmanLab, University of Wisconsin - Madison
+
+Error types intended to organize the different common runtime errors into 
+useful categories where specific handling action pertains to each type. 
+"""
+
+## built-in modules
+from abc import ABC, abstractmethod
+import xml.etree.ElementTree as ET
+
+## local class imports
+from instrument import XMLLoader
+from digitalout import *
```

Comment:
why the star import? The only thing used is in the __main__ call and its one class.